ui/src/pages/frontsite/components/Series/SeriesIdentifiers.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import capitalize from 'lodash/capitalize';
+import get from 'lodash/get';
+import { Table, Divider } from 'semantic-ui-react';
+import { SeparatedList, InfoPopup } from '@components';
+
+export const SeriesIdentifiers = ({ metadata }) => {
```

Comment:
this is similar case as with Abstract, we could use already existing DocumentIdentifiers, especially because the formatting in the metadata is a bit different and it should be unified

Reply:
There is only `DocumentIdentifiers` for the backoffice and that one uses `MetadataTable`. There is an issue open for creating document identifiers for the frontsite but I could rename the series one and reuse on the document details page.